control-plane/api-gateway/cache/consul_test.go.bak

```diff
@@ -29,108 +29,24 @@ import (
 func Test_resourceCache_diff(t *testing.T) {
```

Comment:
Yep, those are why this is still a WIP PR -- anything `.bak` is a file I still need to take a pass over to get the unit tests working, as I prioritized the acceptance test work to actually vet the overall implementation prior to fixing the unit tests while refactoring.

Reply:
Is that a standard I should know about?

Reply:
No, I specifically wanted to go through file by file since most things were broken due to different initialization functions, nil pointer derefs, api contract changes and the like. This was my simple way of:

1. marking which files I haven't gone through yet, and
2. making sure that `go test` doesn't just immediately blow up

Reply:
Makes sense, I was just curious about the use of "bak" as a standard thing that I might encounter again in the future.

Reply:
You make me feel old 😞 -- it's just an old Unix-y way of calling something a backup file, in this case I just needed a different file extension so that they wouldn't get picked up in test runs, so I opted for "backing them up". 😅

Reply:
You are not old! You are wise!